Lib/calendar.py

```diff
@@ -872,10 +872,6 @@ def main(args=None):
 
     options = parser.parse_args(args)
 
-    if options.locale and not options.encoding:
```

Comment:
Why was this removed? I think we should keep it that way

Reply:
Maybe some systems have non utf8 encoding and their locale doesn't support utf8

Reply:
This is a side effect of me previously specifying the encoding='utf-8'. Now following review suggestion to use utf-8 by default when it's None, there's no need to modify this part anymore

The comment below is for the same reason, it's my lack of thoughtfulness